### DISCUSSION TITLE
How to restart a simulation with a new layer of mesh added?

### DISCUSSION THREAD

**Discussion by YYang-farm** [idaholab/moose#16971] [Q&A Modules: General] [answered]
Dear MOOSE team as well as users:
I am recently considering a multilayer phase-field simulation in moose. After the simulation of one layer, I want to add a new layer of mesh, then restart the simulation (and I hope the new mesh will be with the same domain label as the old one, meaning no interface between meshes). Could anybody share with me the possible method to achieve this in moose or any alternatives?
I really appreciate any help you can provide.

By the way, I have also tried using **SolutionUserObject** to map the result of the old layer to a larger mesh (containing two layers) as the IC, which never succeeded since **SolutionUserObject** kept returning "Failed to access the data" for the points located out of the old mesh.

**Comment by GiudGiud** [comment, marked as answer]
Hello

Has this been solved?
If not,
Maybe have the entire mesh in the initial calculation and block restrict the variables / kernels, and load the initial solution only in the relevant blocks. 
Another option would be to use (maybe modify) the control system to activate kernels in more and more blocks as the simulation progresses. 

SolutionUserObject and checkpoint restart will struggle with having different meshes. The meshes must be the same in the two simulations

Guillaume
